FlashSpace/Workspaces/WorkspaceManager.swift

```diff
@@ -16,6 +16,7 @@ final class WorkspaceManager: ObservableObject {
 
     private(set) var activeWorkspace: [DisplayName: Workspace] = [:]
     private(set) var lastWorkspaceActivation = Date.distantPast
+    private(set) var mostRecentWorkspace: [DisplayName: Workspace] = [:]
```

Comment:
Instead of creating a new property like PR currently does, we could change the `activeWorkspace` property to `Subject` to access the old value.

Reply:
I'm not sure how `Subject` would help, but I think the solution you provided is good :)

Reply:
Yeah, If in the future there are additional situations where it would be beneficial to have `activeWorkspace` available as a `Stream` in addition to this feature, it would be nice to change it then. 😄